### ISSUE TITLE
Multiple problems with v1.0.3

### ISSUE THREAD

**Issue by vepth** [FearNixx/vigilant-bans#11]
Hi!

I'm trying to use v1.0.3 for broadcasting. When I install everything, start the LCU Connector and start EU Layout, I get the following error:
```
'react-scripts' is not recognized as an internal or external command, operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! layout-volu-europe@0.1.0 start: `react-scripts start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the layout-volu-europe@0.1.0 start script.                                                           
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.                      
npm WARN Local package.json exists, but node_modules missing, did you mean to install?
npm ERR! A complete log of this run can be found in:                                                                    
npm ERR! D:\vigilant-bans-1.0.3\bin\cache\npm_cache\_logs\2022-02-03T14_45_31_265Z-debug.log                        
[Thread-4] INFO de.fearnixx.lolbanpick.proc.nodejs.NPMProcessRunner - Process finished. 
```

Log https://pastebin.com/UN13Fgnb

Also, is it possible to start the program without initializing installation again but instantly go to start LCU Connector and EU Layout?

One more thing, if I replace lol-pick-ban-ui-master with one from https://github.com/RCVolus/lol-pick-ban-ui (in which I made my custom EU Layout), I get the following error:
https://pastebin.com/da04Ui3u

Explanation: 
- rows 1-3 cannot reinstall because I replaced lol-pick-ban-ui-master with my version (only eu layout changed and npm installed with newest node.js installed from their website). 
- rows 4-8 starting backend with no problems
- rows 9-20 starting eu layout with errors

Thanks for help in advance!

**Comment by MarkTwoFive** [comment]
Hi! 👋 

first of all, the tool downloads and installs the lastest version of the [master branch of the volunteers tool](https://github.com/FearNixx/vigilant-bans/blob/master/src/main/java/de/fearnixx/lolbanpick/Constants.java#L22).  
So the result shouldn't be any different to when you loaded it from GitHub yourself.  

Secondly, after the installation is done once, it does not start again. The application actually only checks whether the directories for it's dependencies exist or not.  

Thirdly, it appears that the Node-Version I hard-coded this tool to use is now no longer the recommended Version for the ban-pick tool. (v14 != v16) So that might need some patching.  
  
And lastly, the dependency error you're reporting here sounds like ``react-scripts`` is now [used directly](https://github.com/RCVolus/lol-pick-ban-ui/blob/master/layouts/layout-volu-europe/package.json#L17) by the layout server. That might mean that how this is set up, a global installation of that NPM package might be required. So the local one that is installed via ``npm install`` is not recognized by NPM.

**Comment by MarkTwoFive** [comment]
The later two notes would require patches to vigilant-bans (and possibly the volunteers' project).  
Currently I am not actively working on this tool and I cannot really estimate when and if I get around to making those changes.

**Comment by vepth** [comment]
Hey Mark!

Thanks for your answer, I'm really happy you made time to help me with this issue! It's really strange that release on my PC installs everything again every time. I'll test a bit more your second and last note to see if I can figure something out :)

**Comment by vepth** [comment]
Also, for the third note (node.js version related one), is it possible for me to use a temporary fix to make the program use v16?

**Comment by vepth** [comment]
Here is the video of everything installing again and again even after everything is in cache folder https://youtu.be/LZRCGvmr-3o

**Comment by MarkTwoFive** [comment]
As you might have or not have noticed. I've published version 1.0.4 of vigilant bans.
It takes care of these required changes and _should_ now work as intended.

If you are still interested, you might try that release. In case one of your issues persists, please open a new issue.
